### PR TITLE
pt_PT

### PR DESCRIPTION
**Pull request by hineios**
Added translation for European Portuguese

### CODE REVIEW

File: pt_PT.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
+{
+  "__LANGUAGE_INFO": {
+    "name": "Português",
+    "author": "Fábio <Hineios> Almeida",
+    "version": "1.0.0"
+  },
+  "File": "Ficheiro",
+  "New...": "Novo...",
+  "Project": "Projecto",
+  "Asset": "Asset", //I can't find a translation that suits this...
```
Comment (Bizarrus) [inline pt_PT.json]: Do not add Comments like this.

File: pt_PT.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  "Properties": "Propriedades",
+  "State": "Estado",
+  "Rember...": "Rember...",
+  "Save the changes before closing?": "Guardar alterações antes de Sair?",
+  "Cancel": "Cancelar",
+  "General": "Geral",
+  "Shortcuts": "Atalhos",
+  "Languages": "Línguas",
+  "Interval:": "Intervalo:",
+  "<u>W</u>orkspace:": "(<u>W</u>) Área de Trabalho:", //I believe the underline is for shortcut purposes
```
Comment (Bizarrus) [inline pt_PT.json]: You can use another character, not only `W`; Please update it for your language specific character. If you dont know, which is a "default", check eclipse IDE or other for a sample.

File: pt_PT.json [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  "Section A": "Secção A",
+  "Section B": "Secção B",
+  "Section C": "Secção C",
+  "Available Languages": "Línguas Disponíveis",
+  "Seconds": "Segundos",
+  "Path to Don't Starve Together...": "Caminho para Don't Starve Together...",
+  "&copy 2017 Adrian Preuß<br />All Rights Reserved.": "&copy 2017 Adrian Preuß<br />Todos os direitos reservados.",
+  "Steam Workshop": "Steam Workshop",
+  "Loading Workshop...": "Carregando Workshop...",
+  "<u>S</u>earch:": "(<u>S</u>) Procurar:", //once again, shortcuts
```
Comment (Bizarrus) [inline pt_PT.json]: See above.

File: pt_PT.json [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  "Version:": "Versão:",
+  "Electron:": "Electron:",
+  "Render Engine:": "Mecanismo the Renderização:",
+  "CPU:": "CPU:",
+  "Network Interfaces:": "Interfaces de Rede:",
+  "Operating System": "Sistema Operativo",
+  "Platform:": "Platforma:",
+  "Release:": "Lançamento:",
+  "Architecture:": "Arquitectura:",
+  "Endianness:": "Extremidade:", //according to Wikipedia
```
Comment (Bizarrus) [inline pt_PT.json]: `Endianness` is `Little Endian` or `Big Endian`, for sample.

**Review by Bizarrus — CHANGES_REQUESTED** [review]
Please fix the comments

File: pt_PT.json (hunk 1 above)
Comment (hineios) [reply pt_PT.json]: The last commit has no comments...

File: pt_PT.json (hunk 4 above)
Comment (hineios) [reply pt_PT.json]: I double checked and the correct term is that one.

**Pull request merged**